### ISSUE TITLE
Difficulty running the example

### ISSUE THREAD

**Issue by monokh** [MetaMask/mascara#9]
I’ve been trying to get this running and facing some difficulties. The first was that the default host path was `zero.metamask.io` so i raised this PR #8 

After using the override function, I wasn't able to get the mascara UI appearing. When the injector runs, it fails to start the service worker - possibly due to `background.js` being quite large? The UI popup happens however it also fails as the service worker wasn't running. I created this codepen to demonstrate: https://codepen.io/anon/pen/ZaWNoy

It would be really good if we could have a live demo to try out

**Comment by monokh** [comment]
Realised i hadn't saved overriding of the host... The correct codepen is: https://codepen.io/anon/pen/gXMrXy

Errors are:
```
The request to fetch the script was interrupted.
wallet.metamask.io/background.js Failed to load resource: net::ERR_FAILED
proxy.js:66 Uncaught (in promise) DOMException: Failed to register a ServiceWorker: The request to fetch the script was interrupted.
```
I'm on a mac with chrome 61

**Comment by frankiebee** [comment]
@monokh the popup needs a user action in order to appear. So if you add a button the pop up will be their. Still trying to figure out a work around for this.

**Comment by monokh** [comment]
@frankiebee by user interaction, do you mean some sort of RPC call? In the codepen, to simulate user interaction, I have put a `sendTransaction` that triggers mascara popup fine. The issue seems to be onload when mascara calls the proxy and creates a service worker, it fails

**Comment by Pipeliner** [comment]
Same error loading background.js as @monokh's.
Any news or workarounds on this one?

**Comment by digitaldonkey** [comment]
I got some [an example](https://github.com/digitaldonkey/mascara_wrapper) with a little more use feedback implemented. It is basically an dapp launcher using mascara.

Maybe I don't really understand how it is supposed to work, but the most confusing for me is that `maybeTriggerPopup()` is bound to the window object. 

Shouldn't I be able to create my own way how the user triggers a transaction? 

I would be happy for some review feedback. Consider this incomplete and a evaluation of how to have a common way to integrate Mascara and and TX signers.

**Comment by pranay01** [comment]
I face the same problem. Any resolution or way to navigate this issue?

**Comment by Densi1** [comment]
Yes,  uninstall metamask through chrome apps. then reinstall metamask from metamask.io with your original 12 word seed. then reinput token contract for your token. If that does not fix it, then uninstall chrome and reinstall it. Then uninstall metamask and reinstall. should be ok , if not contact support@metamask.io

**Comment by pranay01** [comment]
@Densi1  I was looking a way for a way to make Metamask Mascara work. The process which you mentioned will install metamask extension right?
